crates/rocksdb/src/db_manager.rs

```diff
@@ -161,8 +161,7 @@ impl RocksDbManager {
 
         let path = db_spec.path.clone();
         let wrapper = Arc::new(RocksDb::new(self, db_spec, db.clone()));
```

Comment:
Do we still need the `db.clone()` call if we no longer return `db` but the `wrapper` instead?

Reply:
fixed.